snoop/data/_file_types.py

```diff
@@ -4,11 +4,17 @@
     'text/plain': 'text',
     'text/html': 'html',
     'application/x-hush-pgp-encrypted-html-body': 'html',
+    'application/xhtml+xml': 'html',
+    'application/xml': 'html',
+    'text/xml': 'html',
```

Comment:
leave unmapped, as stated in previous comment

Reply:
Changed it.